building/trusted_applications.rst

```diff
@@ -403,18 +402,23 @@ The build script will do two things:
 
 * | It will generate `.stripped.elf` files of the in-tree TAs and sign them with the dummy key
   | pointed to by ``TA_SIGN_KEY``, thus creating `.ta` files. Note that the generated `.ta` files are
-  | not to be used as they are not compatible with the public key embedded into the OP-TEE core image. 
-  
-Note that the rest of the steps below use `.stripped.elf` files, either generated by the compilation of
-the TA DEV KIT (the in-tree TAs) or by generated the user externally.
+  | not to be used as they are not compatible with the public key embedded into the OP-TEE core image.
+
+2. Optionally, build any external TA up to the point so the corresponding `.stripped.elf` file is 
```

Comment:
Building the file to sign is not optional...

Reply:
Fixed.